website/client/src/assets/scss/spacing.scss

```diff
@@ -0,0 +1,387 @@
+.m {

```

Comment:
You probably already know about them but bootstrap has its own spacing utilities https://getbootstrap.com/docs/4.5/utilities/spacing/ which we already use across the code. And I think can be customized changing some of the theme variables. Are these new ones really necessary?

Reply:
It was first created to fix some issues because of the bootstrap margins - we'll try to check the bootstrap's margins again , maybe it'll "just" work now ^^ - if so , this `spacing.scss` file will be removed once all paddings are replaced.